.github/workflows/validate-changelog.yml

```diff
@@ -0,0 +1,17 @@
+name: Validate changelog.json
+
+on:
+  pull_request:
+    paths:
+      - "changelog.json"
+
+jobs:
```

Comment:
let's add a comment to the PR when this fails, pointing to the README.md documentation for the schema